wgsl/index.bs

```diff
@@ -677,36 +677,37 @@ type.
 </pre>
 * type must be `f32`, `i32` or `u32`
 
-### Read-only Storage Texture Types ### {#texture-ro}
+### Storage Texture Types ### {#texture-storage}
 
 A <dfn noexport>read-only texture</dfn> is a texture from which individual texels
-may be read, without the use of a sampler.
-See [[#texture-builtin-functions]].
+may be read, without the use of a sampler. A <dfn noexport>write-only texture</dfn>
+is a texture to which individual texels may be written. See [[#texture-builtin-functions]].
 
 <pre class='def'>
-`texture_ro_1d<image_storage_type>`
+`texture_storage_1d<read|write, image_storage_type>`
```

Comment:
This read|write is meta-syntax that's not explained anywhere.
Instead, write two lines:

> texture_storage_1d<read, image_storage_type>
> texture_storage_1d<write, image_storage_type>

Reply:
done